src/openforms/authentication/utils.py

```diff
@@ -106,6 +106,30 @@ def get_cosign_login_url(request: Request, form: Form, plugin_id: str) -> str:
     return auth_page.url
 
 
+def same_cosigner(auth_attribute: str, auth_value: str, submission: Submission) -> bool:
+    from openforms.submissions.utils import is_same_login_attribute, is_same_login_value
+
+    if not submission.is_authenticated:
+        logger.info(
+            "cosign_not_allowed",
+            reason="missing_auth_info",
+            submission_uuid=submission.uuid,
+        )
```

Comment:
This logging line can move to the form validation.

Also, make sure you stringify the UUID otherwise you get the `repr` in logs:

```suggestion
        logger.info(
            "cosign_start_blocked",
            reason="cosigner_same_as_submitter",
            submission_uuid=str(submission.uuid),
        )
```

I've also changed the event to `cosign_start_blocked`, which is typically the form you want to use for the event name - describe the observable effect as the end-user experiences it, if possible.